api/v1alpha1/storageconsumer_types.go

```diff
@@ -62,6 +70,8 @@ type StorageConsumerStatus struct {
 	CephResources []*CephResourcesSpec `json:"cephResources,omitempty"`
 	// Timestamp of last heartbeat received from consumer
 	LastHeartbeat metav1.Time `json:"lastHeartbeat,omitempty"`
+	// Information of storage client received from consumer
+	Info ClientInfo `json:"info,omitempty"`
```

Comment:
- I thought about it earlier, clientinfo->clientinfospec and info->clientinfo, however I felt this info can't be named like ceph/csi but generic so settled with just `info`
- as for not using `client` in var name, I felt it's implicit that this info is from client, like we'll be adding more fields in there
- having said above, I can change if I get similar comment

Reply:
I agree there is some confusion here. What would you make out of `StorageConsumer.Status.Info`?
Don't it make more sense to have `StorageConsumer.Status.ClientInformation`? but even then why `information`? it describe something that is very static and we might want to piggy back additional non status values

Reply:
Ya, naming is hard, thinking about it again, this captures the latest data that was sent by client but I can't think any specific var to name this.

Reply:
I'm following the existing notion, name of var as json tag rather than the type name, ex:

```
CephResources []*CephResourcesSpec `json:"cephResources,omitempty"`
```

Reply:
done.